src/app/analytics/analytics.factory.js

```diff
@@ -537,9 +537,15 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
               category: cartItem.designationType.toLowerCase(),
               variant: cartItem.frequency.toLowerCase(),
               quantity: '1',
-              dimension3: cartItem.frequency.toLowerCase() === 'single' ? 'one-time' : 'recurring',
+              dimension1: localStorage.getItem('gaDonorType'),
+              dimension2: localStorage.getItem('gaDonorAcct') || undefined,
+              dimension3: cartItem.designationType.toLowerCase(),
```

Comment:
This looks like dimension 8

Reply:
Well, maybe not...But I don't know what "Donation Type" really means.

Reply:
Do you mean the old dimension 8? Brenton originally created the first three dimensions I added(dimension3, 4, 8) but then Lauren went through and remapped the values so things got switched around.

Reply:
Donation Type does not scream Designation Type to me. But maybe that's what they meant.